app/Http/Controllers/ProfileController.php

```diff
@@ -10,12 +10,14 @@
 class ProfileController extends Controller
 {
     // Gets profile update page
+
```

Comment:
It should be  a multiline comment and there is no need for space between the comments and the method. Take care of that for everything in this file